wled00/wled14_colors.ino

```diff
@@ -17,6 +17,29 @@ void colorFromUint32(uint32_t in, bool secondary)
   }
 }
 
+//load a color without affecting the white channel
+void colorFromUint24(uint32_t in, bool secondary = false)
+{
+  if (secondary) {
+    colSec[0] = in >> 16 & 0xFF;
+    colSec[1] = in >> 8  & 0xFF;
+    colSec[2] = in       & 0xFF;
+  } else {
+    col[0] = in >> 16 & 0xFF;
+    col[1] = in >> 8  & 0xFF;
+    col[2] = in       & 0xFF;
+  }
+}
```

Comment:
Added for setting RGB colors without changing the white channel --- function is used in WLED20_ir.ino to get white with different color tones.